polygeist/tools/cgeist/Test/CMakeLists.txt

```diff
@@ -15,7 +15,6 @@ list(APPEND MLIR_CLANG_TEST_DEPS
   llvm-config 
   llvm-dis
   llvm-as
-  mlir-translate
```

Comment:
We still need that for host-raising tests

Reply:
which tests? I tried to grep `mlir-translate` in the `polygeist` folder, but found nothing.

Reply:
True, then that is enough :+1: